howto_docker.md

```diff
@@ -0,0 +1,58 @@
+# How to use Docker and Openfisca
+
```

Comment:
Why would we want to use OpenFisca with Docker ?

From ["How to use OpenFisca on your browser"](https://github.com/openfisca/openfisca-doc/blob/master/howto-web-no-local-install.md):

> If you want to use OpenFisca for running a simulation or evaluating the impact of a reform, without installing anything in your computer, these online services allow you to run OpenFisca directly on your browser, no installation required: repl.it, python anywhere and jupyterlab.

That paragraph tells the reader why she's here and why she should (or not) care about the content of the page. Are we solving a problem for them?

It follows the structure _When you want to [do something], but you can't because of [impediment], know that there is [solution to the impediment] ._

I couldn't stress enough how important that is.